pyspecProcScripts/integral_w_error.py

```diff
@@ -18,6 +13,9 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
     error_path: dict
                 Dictionary of all coherence pathways that are 
                 not the signal pathway.
```

Comment:
don't do this -- rather pass the pathways you DON'T want used for the error -- I had written code for this in one of the examples using some set - based math, so you should be able to reuse that.

Reply:
we decided that we don't want to change this,  but we should make a note for the user to look for examples that contain the variable `excluded_pathways`, so that they can easily specify excluded (rather than  included) pathways

Reply:
when I say "make a note" I mean to add to the docstring here